invocation-count-measurers/src/bench/scala/org/scalameter/examples/ListBenchmark.scala

```diff
@@ -0,0 +1,29 @@
+package org.scalameter.examples
+
+import org.scalameter.api._
+import org.scalameter.execution.invocation.InvocationCountMatcher
+import org.scalameter.execution.invocation.InvocationCountMatcher.{MethodMatcher, ClassMatcher}
+
+
+object ListBenchmark extends PerformanceTest.Microbenchmark {
+  val sizes = Gen.range("size")(300000, 1500000, 300000)
+
+  val lists = for {
+    size <- sizes
+  } yield (0 until size).toList
+
+  performance of "Option" in {
+    measure method "allocations" in {
+      using(lists) config (
+        exec.independentSamples -> 1 // we rather want one JVM instance since this measurer is deterministic
```

Comment:
About the exception, I had a feeling that we'd need to fix this. The regression reporter does some statistics to determine if the performance measurement is a regression, and it is undefined for 1 measurement.

This means that, in the `Measurer`, we should duplicate the measurement `N` times for deterministic tests, instead of just returning a sequence with one measurement.